readme.md

```diff
@@ -10,16 +10,21 @@
 <a href="https://sonarcloud.io/dashboard?id=LycheeOrg_Lychee-Laravel"><img alt="Quality Gate Status" src="https://sonarcloud.io/api/project_badges/measure?project=LycheeOrg_Lychee-Laravel&metric=alert_status"></a>
 </p>
 
-
 #### Help, I'm lost!
 
+Lychee website - [lycheeorg.github.io &#187;](https://lycheeorg.github.io/)
```

Comment:
I would place this bellow the "I was using the version 3,..." 

I don't know what the other think about it. For the rest, no strong opinion. 😃

Reply:
Formatting / order is not something I was too hung up about - just that the links are high up the page. Can set the formatting however you think you'd like it :)